completions/ip

```diff
@@ -282,6 +282,41 @@ _comp_cmd_ip()
                     ;;
             esac
             ;;
+        stats)
+            case "$subcmd" in
+                show)
+                    if [[ $prev == dev ]]; then
+                        _comp_compgen_available_interfaces
+                    elif [[ $prev == group ]]; then
+                        _comp_compgen -- -W "$(
+                            # stats command was added after color, should always have it
+                            $1 -c=never stats help 2>&1 |
```

Comment:
Path to the `ip` command being invoked can theoretically contain spaces, so `"$1"` would be in order here. But I see we're failing that already in other places in the file, want to submit a separate followup PR to fix them all?

Reply:
Sure, fixed it here, and will send a PR for the rest